common/live-streaming.md

```diff
@@ -37,7 +37,7 @@ Use our [Live Streaming Starter Kit](https://www.100ms.live/marketplace/live-str
 
 ![Live streaming starter kit](/docs/v2/live-streaming-starter-kit.png)
 
-1. Make sure that you have [an account with 100ms](https://dashboard.100ms.live/register) and can access the [100ms dashboard](https://dashboard.100ms.live/)
```

Comment:
let's not mix changes in a PR - this doesn't need to be a 30 files change because we've decided to change the casing here

Reply:
sure, removing these changes.